pkg/client/unversioned/daemon_sets_test.go

```diff
@@ -14,7 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package unversioned
+package unversioned_test
```

Comment:
@lavalamp I know it has a special naming. I am not sure why do we want it for this file. And I want to clarify `_test` is not a pkg naming convention that we should use for any cases.